hack/check_pre_req.sh

```diff
@@ -57,6 +57,24 @@ is_installed() {
     fi
 }
 
+check_sysctl() {
+    name="$1"
+    minval="$2"
+    # because the OS running this script might not be the Linux that runs the containers, use `docker run` to interrogate that Linux
```

Comment:
Trying to understand this: Is WSL one possible "Linux that runs the containers"? Any other possibilities?

Reply:
I am not sure I understand the question. Can you restate or elaborate?

Reply:
I'm trying to understand the reason that we need docker instead of running sysctl directly without docker. Trying to find an example of "the OS running this script" and "Linux that runs the containers" are not the same.

Reply:
I see. That's also what I meant by mentioning Windows Subsystem for Linux in the opening comment.

/lgtm

Reply:
There is more to WSL than the guest OS. When I hear "WSL", I think mainly of the Windows machinery that hosts the guest.